doc/deployer/es_initialize.py

```diff
@@ -0,0 +1,132 @@
+from bson import json_util
+import os
+import sys
+import json
+from elasticsearch import Elasticsearch
+from gnowsys_ndf.ndf.models import *
+
+from gnowsys_ndf.settings import GSTUDIO_SITE_NAME
+from gnowsys_ndf.local_settings import GSTUDIO_DOCUMENT_MAPPING
+
+
+##### use the below commented lines if you are working with Python 2.x   #####
+# reload(sys)  
+# sys.setdefaultencoding('UTF8')
+
+es = Elasticsearch("http://elastic:changeme@gsearch:9200", timeout=100, retry_on_timeout=True)
```

Comment:
**Get `username` and `password` from `settings.py` file.**

Sensitive information should not be there in code files and in git. Best practise is to get it from `local_settings.py` or environmental variables.